src/extensions/listExtensions.ts

```diff
@@ -28,21 +27,23 @@ export async function listExtensions(projectId: string): Promise<any> {
     style: { head: ["yellow"] },
   });
   // Order instances newest to oldest.
-  const sorted = _.sortBy(instances, "createTime", "asc").reverse();
+  const sorted = instances.sort(
+    (a, b) => new Date(b.createTime).valueOf() - new Date(a.createTime).valueOf()
+  );
   const formatted: Record<string, string>[] = [];
   sorted.forEach((instance) => {
-    let extension = _.get(instance, "config.extensionRef", "");
+    let extension = instance.config.extensionRef || "";
     let publisher;
     if (extension === "") {
-      extension = _.get(instance, "config.source.spec.name", "");
+      extension = instance.config.source.spec.name || "";
```

Comment:
Do we need to null guard  this with optional chaining?

Reply:
according to the types, no. it all seemed to be there. that's a lot of trust in the types though